Singapore/src/main/java/com/infoshareacademy/service/Menu.java

```diff
@@ -0,0 +1,232 @@
+package com.infoshareacademy.service;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+//stworzyłem metody do obsługi menu w oparciu o kombinację pętli "do while" oraz instrukcji warunkowej "switch"
+
+public class Menu {
+      public static void OpenMenu() {
+        Scanner sc = new Scanner(System.in);
+        int option;      //przerzuciłem zmienne subOption i subSubOption do metody prywatnej OpenSubOption
```

Comment:
```suggestion
        int option;
```
nie musisz tłumaczyć co i dlaczego zrobiłeś